modules/installation-configuration-parameters.adoc

```diff
@@ -654,7 +657,7 @@ a|For example, `sshKey: ssh-ed25519 AAAA..`.
 
 |====
 
-ifdef::aws[]
+ifdef::aws,aws-govcloud,aws-secret,aws-restricted[]
```

Comment:
This part is fine, but above at lines 635 and 638 (`publish: Internal|External`) there is conditional docs that need to need to take these new options into account.